linera-views/tests/map_view_tests.rs

```diff
@@ -106,6 +106,9 @@ async fn run_map_view_mutability<R: RngCore + Clone>(rng: &mut R) {
                             .collect::<Vec<_>>()
                     }
                 };
+                let test_view = view.map.contains_key(&key).await.unwrap();
+                let test_map = new_state_map.contains_key(&key);
```

Comment:
(Or maybe `actual` and `expected`.)

Reply:
I'm afraid I have to disagree here and prefer the `test_view / test/map`.